reana/reana_dev/wiki.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""`reana-dev`'s wiki commands."""
+
+import click
+
+from reana.config import REANA_LIST_DEMO_ALL
+
+GITHUB_REANAHUB_URL = "https://github.com/reanahub"
+CODECOV_REANAHUB_URL = "https://codecov.io/gh/reanahub"
+MAINT_BRANCHES = ["maint-0.7"]
```

Comment:
Note that we have `GIT_DEFAULT_BASE_BRANCH` in `reana/config.py`.  So this variables could go there and use a similar name, perhaps `GIT_SUPPORTED_MAINT_BRANCHES`?  You could also move the other two there.